src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -186,6 +189,15 @@ public int getChangedFiles() throws IOException {
     }
 
     /**
+     * FYI: https://developer.github.com/changes/2013-04-25-deprecating-merge-commit-sha
+     */
+    @Deprecated
+    public String getMergeCommitSha() throws IOException {
```

Comment:
@oleg-nenashev I've checked that if your model has an attribute and is not mapped in the JSON, its `get()` method returns `null`. I think that is ok.